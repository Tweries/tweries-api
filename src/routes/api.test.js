const api = require('./api');

test('api', () => {
  expect(api.stack).toMatchSnapshot();
});
