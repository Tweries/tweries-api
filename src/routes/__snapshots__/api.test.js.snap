// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`api 1`] = `
Array [
  Layer {
    "handle": [Function],
    "keys": Array [],
    "name": "bound dispatch",
    "params": undefined,
    "path": undefined,
    "regexp": /\\^\\\\/v2\\\\/health\\\\/\\?\\$/i,
    "route": Route {
      "methods": Object {
        "get": true,
      },
      "path": "/v2/health",
      "stack": Array [
        Layer {
          "handle": [Function],
          "keys": Array [],
          "method": "get",
          "name": "corsMiddleware",
          "params": undefined,
          "path": undefined,
          "regexp": /\\^\\\\/\\?\\$/i,
        },
        Layer {
          "handle": [Function],
          "keys": Array [],
          "method": "get",
          "name": "getHealthHandler",
          "params": undefined,
          "path": undefined,
          "regexp": /\\^\\\\/\\?\\$/i,
        },
      ],
    },
  },
  Layer {
    "handle": [Function],
    "keys": Array [],
    "name": "corsMiddleware",
    "params": undefined,
    "path": undefined,
    "regexp": /\\^\\\\/v2\\\\/tweet\\\\/\\?\\(\\?=\\\\/\\|\\$\\)/i,
    "route": undefined,
  },
  Layer {
    "handle": [Function],
    "keys": Array [],
    "name": "useTweetHandler",
    "params": undefined,
    "path": undefined,
    "regexp": /\\^\\\\/v2\\\\/tweet\\\\/\\?\\(\\?=\\\\/\\|\\$\\)/i,
    "route": undefined,
  },
  Layer {
    "handle": [Function],
    "keys": Array [],
    "name": "corsMiddleware",
    "params": undefined,
    "path": undefined,
    "regexp": /\\^\\\\/v2\\\\/tweetstorm\\\\/\\?\\(\\?=\\\\/\\|\\$\\)/i,
    "route": undefined,
  },
  Layer {
    "handle": [Function],
    "keys": Array [],
    "name": "multerMiddleware",
    "params": undefined,
    "path": undefined,
    "regexp": /\\^\\\\/v2\\\\/tweetstorm\\\\/\\?\\(\\?=\\\\/\\|\\$\\)/i,
    "route": undefined,
  },
  Layer {
    "handle": [Function],
    "keys": Array [],
    "name": "useTweetstormHandler",
    "params": undefined,
    "path": undefined,
    "regexp": /\\^\\\\/v2\\\\/tweetstorm\\\\/\\?\\(\\?=\\\\/\\|\\$\\)/i,
    "route": undefined,
  },
]
`;
